src/subcommands/deploy.ts

```diff
@@ -0,0 +1,306 @@
+// Copyright 2021 Deno Land Inc. All rights reserved. MIT license.
+
+import { fromFileUrl, join, normalize, Spinner, wait } from "../../deps.ts";
+import { error } from "../error.ts";
+import { API, APIError } from "../utils/api.ts";
+import { ManifestEntry } from "../utils/api_types.ts";
+import { parseEntrypoint } from "../utils/entrypoint.ts";
+
+const help = `deployctl deploy
+Deploy a script with static files to Deno Deploy.
+
+To deploy a local script:
+  deployctl deploy --project=helloworld main.ts
+
+To deploy a remote script:
+  deployctl deploy --project=helloworld https://deno.com/examples/hello.js
+
+To deploy a remote script without static files:
+  deployctl deploy --project=helloworld --no-static https://deno.com/examples/hello.js
```

Comment:
`--no-static` flag isn't working at the moment. It's passed to `deploy(opts)` but not used to condition static uploads.

Reply:
unrelated nit: an example that uses --include/--exclude might be helpful.